test/jasmine/tests/plot_interact_test.js

```diff
@@ -6,7 +6,15 @@ var Lib = require('@src/lib');
 var createGraphDiv = require('../assets/create_graph_div');
 var destroyGraphDiv = require('../assets/destroy_graph_div');
 var customMatchers = require('../assets/custom_matchers');
+var mouseEvent = require('../assets/mouse_event');
+var selectButton = require('../assets/modebar_button');
 
+var MODEBAR_DELAY = 500;
```

Comment:
@etpinard this is needed as the callback turns out to be deferred - the tests pass if I set it to 0, but fail if I unroll the setTimeouts into the main loop (callback has been called 0 times).